docs/config-fcos-v1_5-exp.md

```diff
@@ -208,6 +208,11 @@ The Fedora CoreOS configuration is a YAML document conforming to the following s
     * **_devices_** (list of strings): the list of whole-disk devices (not partitions) to include in the disk array, referenced by their absolute path. At least two devices must be specified.
 * **_extensions_** (list of objects): the list of additional packages to be installed.
   * **name** (string): the name of the package.
+* **_grub_** (object): describes configuration changes for GRUB
```

Comment:
Hmm, maybe something like `describes the desired GRUB bootloader configuration.`?  `Changes` is imperative; we try to use declarative language in the spec.